src/service-worker.js

```diff
@@ -17,11 +17,19 @@
  * See https://goo.gl/2aRDsh
  */
 
-importScripts("/workbox/workbox-sw.js");
-
-workbox.setConfig({
-  modulePathPrefix: "/workbox/",
-});
+if (process.env.NODE_ENV === "development") {
+  importScripts(
+    "https://storage.googleapis.com/workbox-cdn/releases/4.3.1/workbox-sw.js",
+  );
```

Comment:
we don't have any dev workbox scripts so I'm just pulling it from CDN (same version)